rockcraft.yaml

```diff
@@ -7,36 +8,42 @@ description: |
   This is the Canonical Identity platform login UI used for connecting
   Ory Kratos with Ory Hydra.
 license: Apache-2.0
+
 platforms:
   amd64:
 
 services:
   login-ui:
     override: replace
-    command: identity_platform_login_ui
+    command: /usr/bin/identity-platform-login-ui
     startup: enabled
 
 parts:
-  frontend:
+  js-build:
     plugin: npm
-    source: ./ui
+    source: .
     source-type: local
     build-snaps:
-    - node/18/stable
+      - node/18/stable
+    build-packages:
+      - make
+      - git
     override-build: |
-      npm ci --frozen-lockfile
-      npm run build
-  backend:
+      make npm-build
+  go-build:
     plugin: go
-    build-snaps:
-      - go/1.19/stable
-    build-environment:
-      - GOFLAGS: -ldflags=-w -ldflags=-s
-      - CGO_ENABLED: 0
     source: .
     source-type: local
+    build-snaps:
+      - go/1.19/stable
+      - node/18/stable
```

Comment:
probably something that changed since the review, but put all back into one part, the extra split was not really needed, the `make install` with `npm` was the missing part